.github/ISSUE_TEMPLATE/---feature.md

```diff
@@ -1,7 +1,9 @@
 ---
-name: "🎁 Feature request"
+name: "\U0001F381 Feature request"
 about: Suggest an idea for this project
-labels: "Feature 🎁"
```

Comment:
this should go back

Reply:
> Did you mean to add 3 new templates? They all look similar to me

No, I did not :( I think I may have messed up things here... I will close this PR and will try again...I just wanted to create one for Intermittent issues, don't know how the others were added/modified :(